README.md

```diff
@@ -91,7 +91,9 @@ const { posts: postsWithComments } = await firegraph.resolve(firestore, gql`
 
 ### Document References
 
-Right now, we are assuming that `post.author` is a string that matches the ID of some document in the `users` collection. In the future we will leverage Firestore's `DocumentReference` value type to handle both use cases.
+If `post.author` is a `DocumentReference` field, it is considered as a complete path to the document from the root of the database.
```

Comment:
I believe this is just how DocumentReference fields work generally, but it's worth calling out anyway.